overops-functions/src/main/java/com/takipi/udf/microsoftteams/card/MicrosoftTextBuilder.java

```diff
@@ -0,0 +1,70 @@
+package com.takipi.udf.microsoftteams.card;
+
+import com.takipi.api.client.data.event.Location;
+
+import java.util.List;
+
+public class MicrosoftTextBuilder {
+    private StringBuilder stringBuilder = new StringBuilder();
+
+    public MicrosoftTextBuilder add(String text) {
+        stringBuilder.append(text);
+
+        return this;
+    }
+
+    public MicrosoftTextBuilder addBold(String text) {
+        stringBuilder.append("<strong>").append(text).append("</strong>");
+
+        return this;
+    }
+
+    public MicrosoftTextBuilder addQuote(String text) {
+        stringBuilder.append("<blockquote>").append(text).append("</blockquote>");
+
+        return this;
+    }
+
+    public MicrosoftTextBuilder addHighlighted(String text) {
+        stringBuilder.append("<pre>").append(text).append("</pre>");
+
+        return this;
+    }
+
+    public MicrosoftTextBuilder addHighlightedQuote(String text) {
+        stringBuilder.append("<blockquote><pre>").append(text).append("</pre></blockquote>");
+
+        return this;
+    }
+
+    public MicrosoftTextBuilder addLink(String link, String nameOfLink) {
+        stringBuilder.append("<a href=\"").append(link).append("\">").append(nameOfLink).append("</a>");
+
+        return this;
+    }
+
+    public MicrosoftTextBuilder addBoldLink(String link, String nameOfLink) {
+        stringBuilder.append("<a href=\"").append(link).append("\">").append("<strong>").append(nameOfLink).append("</strong>").append("</a>");
+
+        return this;
+    }
+
+    public MicrosoftTextBuilder addEnter() {
+        stringBuilder.append("\n\n");
+
+        return this;
+    }
+
+    public String build() {
+        return stringBuilder.toString();
+    }
+
+
+    public MicrosoftTextBuilder addArray(List<Location> stack_frames, String prefix) {
```

Comment:
While the JSON is being serialized with underscores, method parameters should be in camel case (i.e `stackFrames` in this case)

Reply:
done, this class will not be serialized it just until class to create MicrosoftTeams Card text pseudo Html markup